sui/typescript/dapp_kit_example/.env

```diff
@@ -0,0 +1,4 @@
+ALL_SERVICES_TESTNET_ACCESS_KEY = 'Super key with access to Shinami node, gas station, and wallet services. Only used on backend.'
+EXAMPLE_MOVE_PACKAGE_ID = '0xd8f042479dcb0028d868051bd53f0d3a41c600db7b14241674db1c2e60124975'
+USER123_WALLET_ID = 'Static wallet id for one specific user. In a real app you would look up the walletId for a given userId'
+USER123_WALLET_SECRET = 'Static wallet secret for one specific walletId.'
```

Comment:
Should these be commented out by default (maybe except `EXAMPLE_MOVE_PACKAGE_ID`)?

Reply:
Wasn't sure, since running the app requires you to set them all. But I've done that and updated the README accordingly.